mimic/test/test_heat.py

```diff
@@ -84,6 +84,31 @@ def create_stack(self, stack_name, tags=None):
         data = self.get_responsebody(resp)
         return data
 
+    def update_stack(self, stack_name, stack_id, resp_code=202):
+        """
+        Request stack update and assert that the response matched the one
+        provided.
+        """
+        req_body = {'foo': 'bar'}
+        req = request(self, self.root, b"PUT",
+                      '%s/stacks/%s/%s' % (self.uri, stack_name, stack_id),
+                      body=json.dumps(req_body).encode("utf-8"))
+        resp = self.successResultOf(req)
+        self.assertEqual(resp.code, resp_code)
+
+    def check_stack(self, stack_name, stack_id, resp_code=201,
```

Comment:
This is called below to perform other actions. How about renaming it to `perform_stack_action` instead?